### PR TITLE
Add/sidebar

### PR DESCRIPTION
**Pull request by cristianUrbina**
Add sidebar feature

### CODE REVIEW

File: src/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  flex-flow: wrap column;
+  justify-content: center;
+  align-items: center;
+}
+
+.sidebar .material-icons {
+  margin: 10px;
+}
+
+.sidbar a {
```
Comment (erickgl1003) [inline src/components/sidebar/sidebar.css]: Perhaps you could change sidbar name to something more accurate?

**Review by erickgl1003 — APPROVED** [review]
Sidebar design meets expectations:

- [ ✓ ] Design constraints taken in account.
- [ ✓ ] Proper standardized code style is used.
- [ ✓ ] No errors.

Notes:

- [ ? ] Change sidbar name?

File: src/components/sidebar/sidebar.css (hunk 1 above)
Comment (cristianUrbina) [reply src/components/sidebar/sidebar.css]: Any suggestions?

Comment (erickgl1003) [reply src/components/sidebar/sidebar.css]: sidebar

**Pull request merged**